core-libs/setup/ssr/optimized-engine/optimized-ssr-engine.ts

```diff
@@ -39,6 +47,7 @@ export type SsrCallbackFn = (
 export class OptimizedSsrEngine {
   protected currentConcurrency = 0;
   protected renderingCache = new RenderingCache(this.ssrOptions);
+  protected logger: ExpressServerLogger;
```

Comment:
[MEDIUM]

What about keeping it private?
```suggestion
  private logger: ExpressServerLogger;
```